### PR TITLE
Converting from javax.* to jakarta.*

### LINKED ISSUE #266: Need updated spec-version-maven-plugin
The current version has a hard-coded requirement that the groupId starts with `javax`. That breaks the build as we are using a groupId of `jakarta`

### PR DESCRIPTION
**Pull request by joakime**
Signed-off-by: Joakim Erdfelt <joakim.erdfelt@gmail.com>

Closes #266

### CODE REVIEW

File: spec/pom.xml [hunk 1]
```diff
@@ -58,8 +58,8 @@
                         <configuration>
                             <rules>
                                 <requireJavaVersion>
-                                    <version>[1.8.0,1.9.0)</version>
-                                    <message>You need JDK8 or lower</message>
+                                    <version>[1.8.0,)</version>
+                                    <message>You need JDK8 or higher</message>
                                 </requireJavaVersion>
```
Comment (markt-asf) [inline spec/pom.xml]: I don't think this should be part of the package rename.

File: pom.xml [hunk 2]
```diff
@@ -33,12 +33,12 @@
     </scm>
 
     <properties>
-        <api_package>javax.websocket</api_package>
+        <api_package>jakarta.websocket</api_package>
         <spec_version>1.1</spec_version>
         <!--<spec_build>02</spec_build>-->
         <new_spec_version>1.1</new_spec_version>
         <spec_impl_version>1.1.2</spec_impl_version>
```
Comment (markt-asf) [inline pom.xml]: I need to remind myself exactly what these do but my initial expectation is that these version numbers also need to change to reflect the update to 2.0.0.

File: spec/pom.xml (hunk 1 above)
Comment (joakime) [reply spec/pom.xml]: Unfortunately, I had to update the copyright and spec plugins to allow this to work.
Upgrading those plugins triggered this enforcer rule.
So I adjusted the enforcer rule to be consistent to how the other projects behave.

But I agree, it feels way out of place in this PR.
I can make that change a different PR, and then I can modify this PR to not include this change, but that means this PR becomes dependent on another PR (which is something I don't like doing)
You OK with this approach?

File: pom.xml (hunk 2 above)
Comment (joakime) [reply pom.xml]: Yup, those were missed.
Fixing.

File: spec/pom.xml (hunk 1 above)
Comment (markt-asf) [reply spec/pom.xml]: Fair enough. Lets just include it in this PR. Maybe include a note in the commit message?

**Pull request merged**